class EndpointError(Exception):
    """Исключение: Ошибка в адресе."""

    pass


class StatusError(Exception):
    """Исключение: Ошибка в статусе."""

    pass
